src/components/PasswordInput.tsx

```diff
@@ -0,0 +1,84 @@
+import React,{useState} from 'react';
+import { StyleSheet, Text, Image, View, TouchableHighlight, SafeAreaView, Button, Alert, Platform, StatusBar, Dimensions,TextInput} from 'react-native'
+import COLORS from '../constants';
+import { Ionicons } from '@expo/vector-icons';
+import { Feather } from '@expo/vector-icons';
+
+interface PasswordProps{
+    onChangeText: (text: string) => void
+    value:string
+    text:string
+}
+
+
+
+function PasswordInput({text}:PasswordProps) {
+    const [password, setPassword] = useState('')
+    const [showPassword, setShowPassword] = useState(false)
+   
+    
+    const toggleShowPassowrd = () => {
+        setShowPassword(!showPassword)
+    }
+    return (
+        <View style={styles.InputView}>
+            <TextInput style={styles.input}
+                placeholder={text}
+                onChangeText={setPassword}
+                secureTextEntry={!showPassword}
+                value={password}
+                placeholderTextColor={COLORS.Grey}
+                 
+              
+            />
+            <View style={styles.IconView}>
+                 <Ionicons
+                onPress={toggleShowPassowrd}
+                name={showPassword ? "eye" : "eye-off"}
+                size={24}
+                color={showPassword? "black":"grey"}
+                
+                />
+            </View>
+           
+                
+           
+        </View>
+    );
+}
+
+export default PasswordInput;
+
+const styles = StyleSheet.create({
+    input: {
+               
```

Comment:
Kindly remove the blank spaces present in your code base

Reply:
Done